content/1.7/back-office/international/locations/countries/countries-listing.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Countries listing
+weight: 1
+---
+
+# Countries listing
+
+## Call to action 
+
+A call to action buttont at the top right is displayed:
+
+ - **Add new country** - See the [specs of the add edit country]({{<ref "add-edit-country.md">}} "Add edit country") 
+
+### Settings wheel
+
+When clicking on the settings wheel, the following options are displayed:
+
+1. Add new
+
+When pressed redirects to the [specs of the add edit country]({{<ref "add-edit-country.md">}} "Add edit country") 
+
+2. Refresh list
+
+After clicking on it, the page is reloaded. Sorts and searches are kept.
+
+3. Show SQL query
+
+After clicking on it, the SQL query is displayed in a modal. The SQL query takes into account filters and sorts that have been done.
+
+There are two buttons:
+
+Close: After clicking on it, the modal is closed.
+Export to SQL Manager: After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager.
+
+4. Export to SQL Manager
+
+After clicking on it, the SQL query is opened in Advanced parameters > Database > SQL Manager.
+
+The count of the states currently added is displayed on the left - "States - (current number of states added in the shop)".
+
+### Columns
+
+1. ID
+
+It's possible to sort by id asc or desc and search by id.
+
+2. Country
+
+It's possible to sort by country asc or desc and search by country.
+
+3. ISO code
+
+It's possible to sort by ISO code asc or desc and search by ISO code.
+
+4. Call prefix
+
+It's possible to sort by call prefix asc or desc and search by a call prefix.
+
+5. Zone
+
+It's possible to sort zones by asc or desc and search by zones.
+
+6. Enabled
+
+This is a drop-down list for selecting either "yes" or "no" can be searched.
+
+7. Actions
+
+ - **Search** - Button for searching in the list.
+ - **Edit** - When pressed redirects to the edit page of the selected country.
```

Comment:
```suggestion
 - **Edit** - This is the main action and it can be performed by clicking on the line. After clicking on it, the edit country page is displayed.
```

Reply:
Could you add the link of the page ?